### ISSUE TITLE
[Suggestion] Lesser Spirit Tree Saplings

### ISSUE THREAD

**Issue by cybercat5555** [TheCyberBrick/The-Betweenlands#738]
So the new Spirit Trees, both the natural ones and the grown ones, are absolutely beautiful. I can see why why the saplings should be hard to come by (free protection), the the trees themselves look as if they would make great decorative trees, but their rarity make it hard, especially for a tree. So why not add a "Lesser Spirit Tree Sapling" that drops from the leaves of the trees, and can be found as loot? This kind would not have the added protection, as it would not be "possessed", but instead be a normal tree, just with the fancy leaves and bark, and breaking its leaves would not drop Spirit Fruit, but would have a chance of dropping more Lesser Saplings. This would also be useful if/when Spirit Wood planks and related are added, so they could be a viable building block.

TL; DR: why not add a "Lesser Spirit Tree Sapling" that drops from the leaves of other Spirit Trees (sapling-grown or generated), but to balance it out, this version would not be able to attack mobs nor be able to drop Spirit Fruits, only being able to drop more Lesser Saplings. Essentially a purely decorative version of the sapling-grown Spirit Trees.

**Comment by TripleHeadedSheep** [comment]
I get your point, but at the same time I think it is nice to have some rare 'decorational' stuff. I think when the trees become too easy to come by, and thus might become too common of a sight, they lose their special touch.
To me, the rarity of them makes them a bit more beautiful ;)

But that's open to discussion. It's definitely something to consider
